Sources/SPMClientUtils/ClientUtils.swift

```diff
@@ -0,0 +1,78 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2014-2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+// In this file, only import modules that are part of SwiftPMDataModel (or swift-tools-support-core)!
+// This way, we cannot accidentally use unavailable types in the API intended for clients.
+@_exported import PackageGraph
+@_exported import PackageModel
+@_exported import func TSCBasic.tsc_await
```

Comment:
lets avoid this one - we plan to transition to async/await and deprecate it soon

Reply:
I removed `tsc_await`.